scripts/do-release.sh

```diff
@@ -203,7 +203,11 @@ function update_operator_pr() {
         update_go_mod "${target}"
     done
 
-    sed -i -E "s/(.*Version +=) .*/\1 \"${release['version']#v}\"/" "projects/${project}/apis/submariner/v1alpha1/versions.go"
+    local versions_file
+    versions_file=$(grep -l -r 'Default.*Version *=' projects/${project}/ | grep '.go$')
```

Comment:
If we can assume GNU `grep`:

```suggestion
    versions_file=$(grep -l -r --include='*.go' 'Default.*Version *=' projects/${project}/)
```